README.md

```diff
@@ -62,8 +62,9 @@ if (! Electron.isDesktop){
 
 ## Deploy
 
-Copy `YOUR_PROJECT_DIRECTORY/.meteor-electron/final/app-darwin.zip` to a publically-accessible
-location, then set `latestUrl` in `Meteor.settings.electron` to that URL.
+Copy `YOUR_PROJECT_DIRECTORY/.meteor-electron/final/YOUR_APP_NAME.zip` to a publically-accessible
+location, then set `downloadUrl` in `Meteor.settings.electron` to that URL. This URL will be served
+at `/app/latest/download`.
```

Comment:
@rissem this could serve as the basis of a template helper once again if you liked.